src/pwacompat.js

```diff
@@ -139,13 +139,35 @@ function unused() {
     }
   }
 
+  /**
+   * @param {object} icon
+   * @param {string} icon.sizes
+   */
+  function largestSize(icon) {
+    const sizes = icon.sizes.split(' ').map((size) => {
+                                                          if (
```

Comment:
formatting is messed up ;-)

Reply:
Oof, auto-format ran wild just as I switched off the file. Fixed now!